docs/articles/documentation/extending-testcafe/reporter-plugin/README.md

```diff
@@ -81,14 +85,14 @@ export default function () {
 }
 ```
 
-These methods should output the desired information at certain moments during the test run.
-All the required data is provided for these methods through their parameters.
+TestCafe calls these methods during the test run. Use data they receive through their parameters to compose a report.
+See [Reporter Methods](reporter-methods.md) for API description and examples.
 
-To output this information, use [helper methods and libraries](helpers.md).
-TestCafe will mix in the helper methods to the reporter, so that you can access the helper methods by using `this`.
+To output the report, use [helper methods and libraries](helpers.md).
```

Comment:
to output the report? to output?
I am not familiar with this usage

Reply:
Maybe, similar to streams, we can say *write*?

*Use helper methods and libraries to write and format the report.*

Reply:
yes that's great